config/m4/ucm.m4

```diff
@@ -14,7 +15,8 @@ AC_ARG_ENABLE([symbol-override],
 	[enable_symbol_override=yes])
 	
 AS_IF([test "x$enable_symbol_override" == xyes], 
-	[AC_DEFINE([ENABLE_SYMBOL_OVERRIDE], [1], [Enable symbol override])]
+	[AC_DEFINE([ENABLE_SYMBOL_OVERRIDE], [1], [Enable symbol override])],
```

Comment:
This does not disable the hooks. It just set the flag indicating that SYMBOL_OVERRIDE was disabled.